core/app/models/spree/order.rb

```diff
@@ -787,7 +787,7 @@ def after_cancel
     end
 
     def send_cancel_email
-      OrderMailer.cancel_email(self.id).deliver_later
+      OrderMailer.cancel_email(self).deliver_later
```

Comment:
I want to pass an ActiveRecord object through ActiveJob. Is there any reason I shouldn't? I believe it handles serialization well.